app/controllers/api/v1/admin/user/invoices_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::Admin::User::InvoicesController < Api::V1::BaseController
+  before_action :authenticate_user!, :load_user
+
+  def index
+    authorize :invoice, :index?
+    @invoice = @user.timelogs.date_range(Date.parse(invoice_params.to_s).beginning_of_month, Date.parse(invoice_params.to_s).end_of_month)
+    render json: [@invoice, @user, @user.salary_type, @user.salary, @user.count_day_offs_this_month]
+  end
+
+
+  private
+    def invoice_params
```

Comment:
couple things here:
1. if it is `invoice_params` it should have some relation to invoice. In our case - `invoice` root key. If not, we should change the method name. Otherwise, it is confusing.
2. It is a bad practise to modify params. I am suggesting to do the following:
```
def some_awesome_method_name
  params.perming(:date)
end

def formatted_date
  if some_awesome_method_name
    Date.parse(some_awesome_method_name)
  else
    Date.today
  end
end
```